src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/CancellationTokenParametersMustComeLastTests.cs

```diff
@@ -191,6 +191,22 @@ public static void M1(this CancellationToken p1, object p2)
             VerifyCSharp(test);
         }
 
+        [Fact, WorkItem(1816, "https://github.com/dotnet/roslyn-analyzers/issues/1816")]
+        public void NoDiagnosticWhenMultipleAtEndOfParameterList()
```

Comment:
Can we add a unit test to ensure `public static void M1(CancellationToken token1, object p1, CancellationToken token2) { }` is flagged?

Reply:
Sure, done in 0f16ce222a2e5690538c29f38e009d3e8572fcef.